packages/client/src/views/RegisterForm/duplicate/DuplicateFormTabs.test.tsx

```diff
@@ -1167,54 +1173,52 @@ describe('when user is in the register form review section', () => {
     ]
     store.dispatch(setInitialDeclarations())
     store.dispatch(storeDeclaration(declaration))
-    const mock: any = vi.fn()
+
     vi.spyOn(profileSelectors, 'getScope').mockReturnValue(['register'])
 
     const form = await getReviewFormFromStore(store, EventType.Birth)
 
-    const testComponent = await createTestComponent(
-      // @ts-ignore
+    // @TODO
+    const foo = formatUrl(REVIEW_EVENT_PARENT_FORM_PAGE_GROUP, {
```

Comment:
Figure out which URL needs to be given here. I changed it group since it was given as part of match.